backend/plugins/authorization.js

```diff
@@ -65,7 +66,8 @@ export function PlotlyOAuth() {
                       res.setCookie('db-connector-auth-token',
                                     dbConnectorAccessToken, {
                                         'maxAge': getAccessTokenExpiry(),
-                                        'path': '/'
+                                        'path': getSetting('DB_CONNECTOR_URL'),
```

Comment:
Maybe just `WEB_BASE_PATHNAME` would be a simpler setting. I think that `DB_CONNECTOR` is a little bit redundant since the settings implicitly refer to itself anyway. And URL is usually the full URL (`https://domain/pathname`) and we just need the `pathname`